NSS-Spontaneous-Spoonbills/Sprint3
# Payment View / Customer-User Link

## Link to Ticket

Autolink format: hashtag and number, ex: #1094

Closes #
I don't believe any tickets were closed.

```
git fetch --all
git checkout js_nav_payment_view
```

## Description of Proposed Changes

1. Added Payment to the nav bar. *Will refactor to a dropdown later on.
1. Can open payment list view.
1. Refactored auth_view to link customer and user table 
1. Added two lines of code to the settings.py file to allow for static css files to be rendered


## Steps to Test
1. Run virtual machine
1. Remake migrations & migrate
1. faker_factory data not working with my branch,  ⛔️ so don't try it. ⛔️ 
1. Run server
1. Check that you can successfully register a user
1. Open DB Browser to check and see that: auth_user saved the user
1. Check website_customer and see that the remaining information was saved on the table.
1. Check that the id & user_id are the same.
1. Logic to limit navigation view should have remained the same
1. Some basic styling was created for the navbar
1. in DB Browser, create a payment type then create a payment option to the created user
**SQL Script**
```
INSERT INTO website_payment_type VALUES (null, 'Visa');
INSERT INTO website_payment_option VALUES (null, '54321', 1, 1);

```





## Impacted Areas in Application
1. templates/
1. views/
1. urls.py
1. forms/
1. settings.py

**New Files**:

	website/templates/account/payment_option_detail.html
	website/templates/account/payment_option_list.html
	website/templates/account/payment_option_update.html

**Modified Files**:

	website/views/auth_view.py
	website/views/payment_option_view.py
	website/views/product_type_view.py
	website/views/__init__.py
	website/templates/product/Product_Type_List.html
	website/templates/register.html
	website/templates/product/detail.html
	website/templates/product/product_category.html
	website/urls.py
	website/forms/customer_form.py

**Deleted Files**:
**These files were just moved to a different directory**

	website/templates/account_info/payment_option_detail.html
	website/templates/account_info/payment_option_list.html
	website/templates/account_info/payment_option_update.html


## Mentions @username

Tag users that need to review this code

## Testing

No testing was done

## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
